vendor/assets/stylesheets/vendor.scss

```diff
@@ -1 +0,0 @@
-@import "/**/*";
```

Comment:
We no longer use it, and I hope we would not need to as we moved assets to npm/yarn packages